### PR TITLE
Add 0.7 release post

### PR DESCRIPTION
**Pull request by Xaeroxe**
Release 0.7, to be merged after https://github.com/amethyst/amethyst/pull/730

[Rendered](https://github.com/Xaeroxe/website/blob/0.7/src/_posts/2018-05-18-blog.md)

### CODE REVIEW

File: src/_posts/2018-05-18-blog.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+We're happy to announce that we've released Amethyst 0.7.0!
+to [crates.io](https://crates.io)!
+
+There are 124 pull requests in this release.  If you want to see for yourself
+here's a [link][ep]!
+
+[ep]: https://github.com/amethyst/amethyst/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Amerged+merged%3A2017-12-29..2018-05-18
+
+
+## Notable additions
```
Comment (torkleyy) [inline src/_posts/2018-05-18-blog.md:26]: Is this everything that's left from 124 PRs?

File: src/_posts/2018-05-18-blog.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+[#674]: https://github.com/amethyst/amethyst/pull/674
+[#679]: https://github.com/amethyst/amethyst/pull/679
+[#683]: https://github.com/amethyst/amethyst/pull/683
+[#662]: https://github.com/amethyst/amethyst/pull/662
+[#698]: https://github.com/amethyst/amethyst/pull/698
+[#691]: https://github.com/amethyst/amethyst/pull/691
+
+## Calls for participation
+
+We've opened a few issues that we think are good for first time contributors,
```
Comment (torkleyy) [inline src/_posts/2018-05-18-blog.md]: I like that, but I want to have some kind of document that shows you how to get started. It should provide something like an architectural overview and how "participation" works best here.

File: src/_posts/2018-05-18-blog.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+take a look at the issues on this list and see if any are a good fit for you!
+
+* [#714](https://github.com/amethyst/amethyst/issues/714)
+* [#686](https://github.com/amethyst/amethyst/issues/686)
+* [#669](https://github.com/amethyst/amethyst/issues/669)
+* [#582](https://github.com/amethyst/amethyst/issues/582)
+
+
+If you have any suggestions or ideas for the blog, feel free to voice them
+[on GitHub][gh] or [the Gitter chat][gc].
```
Comment (torkleyy) [inline src/_posts/2018-05-18-blog.md]: or Discord ;)

File: src/_posts/2018-05-18-blog.md (hunk 1 above)
Comment (Xaeroxe) [reply src/_posts/2018-05-18-blog.md:26]: I just grabbed stuff from the changelog, but I'll double check tomorrow.

File: src/_posts/2018-05-18-blog.md (hunk 2 above)
Comment (Xaeroxe) [reply src/_posts/2018-05-18-blog.md]: Agreed, can we draft that and link it in CONTRIBUTING.md instead though?  Seems a bit out of scope for this post imo.

File: src/_posts/2018-05-18-blog.md (hunk 1 above)
Comment (torkleyy) [reply src/_posts/2018-05-18-blog.md:26]: I still need to add my stuff I believe. Except somebody else did that?

File: src/_posts/2018-05-18-blog.md (hunk 2 above)
Comment (torkleyy) [reply src/_posts/2018-05-18-blog.md]: Yeah, a dedicated document. Maybe an issue, not sure yet.

File: src/_posts/2018-05-18-blog.md (hunk 1 above)
Comment (Xaeroxe) [reply src/_posts/2018-05-18-blog.md:26]: Wow uh yeah that really is everything worth putting on the blog post imo.  Most of our PRs have been bug fixes and correctness updates here and there.  These are all the major things that impact users.

**Review by torkleyy — APPROVED** [review]

**Review by torkleyy — APPROVED** [review]
bors r+

**Pull request merged**